packages/common/src/directives/ng_optimized_image/image_loaders/netlify_loader.ts

```diff
@@ -90,6 +91,13 @@ function createNetlifyUrl(config: ImageLoaderConfig, path?: string) {
     url.searchParams.set('w', config.width.toString());
   }
 
+  // When requesting a placeholder image we ask for a low quality image to reduce the load time.
+  // If no quality is specified
```

Comment:
Looks like an incomplete sentence.
```suggestion
  // If the quality is specified in the loader config - always use provided value.
```